src/components/wizard/DoneView.tsx

```diff
@@ -62,6 +256,14 @@ class DoneView extends React.Component<DoneProps> {
                 >
                   Set up another
                 </Button>
+                <Button
+                  className={this.props.classes.button}
+                  variant="contained"
+                  size="large"
+                  onClick={() => this.toggleMonitor()}
+                >
+                  Toggle monitor
```

Comment:
change this to `start/stop monitor` depending on if the monitor runs or not. (use `state.monitorRunning`)